packages/sit-onyx/src/types/required.ts

```diff
@@ -0,0 +1,29 @@
+import { computed } from "vue";
+
+const REQUIRED_MARKER = ["optional", "required"] as const;
+export type RequiredMarkerType = (typeof REQUIRED_MARKER)[number];
+
+export type RequiredMarkerProp = {
+  /**
+   * Whether the form input is required.
```

Comment:
```suggestion
   * Whether the form value is required.
```

The prop is also used for non-text-input components like switches etc.

Reply:
I would argue that conceptually these are all kind of inputs. In HTML they are also, just different types of inputs.

Reply:
Sure, I just think that "input" might confuse some users because they are just thinking about a text input and not a checkbox, switch, dropdown etc.

Reply:
changed it